src/main/java/net/greatstart/services/UserService.java

```diff
@@ -67,6 +68,7 @@ public User changeUserPassword(User user, String password) {
     }
 
     public User getUserByEmail(String email) {
+        System.out.println("find by email " + email);

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)